src/utils/helpers.js

```diff
@@ -113,8 +113,11 @@ const utils = {
     return null;
   },
 
-  hasOrderIdRecord(orderId) {
-    const orderIdHash = CRC32.str(orderId.toString());
+  hasOrderIdRecord({ orderId, apiKey }) {
```

Comment:
Does it matter that this change isn't backwards compatible? I guess as long as this helper is only called from the one spot it should be ok.

Reply:
yeah, this is an internal function from what I can tell so I figure a cleaner implementation is better than looping in backwards compatibility but this is also a really old function. Let's get some of the grand sages of the past to give a second review in case I missed something.